src/base/bittorrent/filterparserthread.cpp

```diff
@@ -37,75 +42,120 @@
 
 #include "base/logger.h"
 
+using namespace std::literals;
+
 namespace
 {
-    class IPv4Parser
+    struct IPFilterEntry
+    {
+        std::string_view range;
+        int accessLevel = 0;
+        // We don't care about this
+        // std::string_view description;
+    };
+
+    IPFilterEntry parseDATLine(std::string_view line)
```

Comment:
if `line` doesn't change value through out this function, you should consider adding `const` to it.
The same to parameters of other functions.

Reply:
ok, but does it makes sense to use `const` with a string_view?

Reply:
>but does it makes sense to use const with a string_view?

Of course, just like using `const` on any variable: if you don't intend to modify it further and wish to keep the initialized value/view.